alicloud/extension_drds.go

```diff
@@ -0,0 +1,13 @@
+package alicloud
+
+type InstanceType string
+type DRDSInstancePayType string
+
+const (
+	PrivateType             = InstanceType("PRIVATE")
+	PublicType              = InstanceType("PUBLIC")
+	PrivateType_            = InstanceType("1")
+	PublicType_             = InstanceType("0")
+	DRDSInstancePostPayType = DRDSInstancePayType("drdsPost")
+	DRDSInstancePrePayType  = DRDSInstancePayType("drdsPre")
```

Comment:
I order to keep consist with other resources, please use const `PrePaid` and `PostPaid`

Reply:
fix